doc/getting-started/building-from-source.rst

```diff
@@ -254,15 +246,20 @@ The ``acrn.efi`` will be generated in the ``./hypervisor/build/acrn.efi`` direct
 
 As mentioned in :ref:`ACRN Configuration Tool <vm_config_workflow>`, the Board configuration and VM configuration can be imported from XML files.
 If you want to build the hypervisor with XML configuration files, specify
-the file location as follows:
+the file location as follows (Assuming at the top level of the acrn-hypervisor directory):
 
 .. code-block:: none
 
    $ make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/nuc7i7dnb.xml \
-   SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/nuc7i7dnb/industry.xml FIRMWARE=uefi
+   SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/nuc7i7dnb/industry.xml FIRMWARE=uefi TARGET_DIR=xxx
 
 
-Note that the file path must be absolute. Both of the ``BOARD`` and ``SCENARIO`` parameters are not needed because the information is retrieved from the XML file. Adjust the example above to your own environment path.
+.. note::
+   Both of the ``BOARD`` and ``SCENARIO`` parameters are not needed because the information is retrieved
+   from the XML file.
+   Parameter TARGET_DIR specify a related path of acrn-hypervisor folder to store configuration files which
```

Comment:
What is creating these files? Is that the `make` command?

Should that folder exist? If so, does it need to be populated with any content?

Reply:
The python scripts in acrn-config will create these files. The benefit of usage of this folder is that the VM configurations from external XMLs could be generated and stored in this folder for build so that the original configuration source would not be "polluted".